dvc/repo/experiments/show.py

```diff
@@ -248,8 +248,7 @@ def get_names(repo: "Repo", result: Dict[str, Dict[str, Any]]):
                 rev = baseline
                 if rev == "workspace":
                     continue
-                name = names.get(rev, None)
-            name = name or exact_name[rev]
+            name = names.get(rev, None) or exact_name[rev]
```

Comment:
```suggestion
            name = names.get(rev, exact_name[rev])
```

Reply:
```
$ a = {}
$ b = {"a": 1}
$ b.get("a", a["b"])
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Input In [5], in <cell line: 1>()
----> 1 b.get("a", a["b"])

KeyError: 'b'

$ b.get("a") or a["b"]
Out[6]: 1
```

Expressions inside `get()` will be evaluated while expressions after `or` will not.